### ISSUE TITLE
On the Participant Selector tab, implement separate scroll boxes for the controls and the output.

### ISSUE THREAD

**Issue by samussiah** [SafetyGraphics/safetyGraphics#507]
For instance with a dataset that has dozens of variables it would be nice to be able to scroll through them while viewing the table.

**Comment by jwildfire** [comment]
Makes sense. Think we can do it with a css tweak, @samussiah ? Or will we need to tweak the equisse code somehow?
